eng/pipelines/scripts/generate_overview_from_readme.py

```diff
@@ -41,6 +41,7 @@ def generate_overview(readme_file, version):
         # markdown2.markdown will create html from the readme.md file. The fenced-code-blocks
         # extras being passed into the markdown call is necessary to deal with the embedded
         # code blocks within the readme so they'll displaye correctly in the html
+        readme_content = markdown2.markdown(re.sub(pattern='{@', repl='{{@literal @}', string=readme_content), extras=["fenced-code-blocks"])
```

Comment:
@yiliuTo why are we making change? Literally, the line below this is doing the same thing. That change was checked in by @alzimmermsft with this [PR](https://github.com/Azure/azure-sdk-for-java/pull/12627).

Reply:
@JimSuplizio - if @alzimmermsft 's change is working correctly, how come I am still seeing the java docs issue on this PR ?
https://github.com/Azure/azure-sdk-for-java/pull/12964

Reply:
Yes, this is similar to the PR mentioned, but this is making the check more flexible to cover additional replacing scenarios. Made a commit to the PR to make the regex match all `@`s and replacing them with the Javadoc safe `{@literal @}`.